### PR TITLE
Added link to Gatsby Style Guide near bottom

### PR DESCRIPTION
**Pull request by shannonbux**

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit ee5ee2ce2effb858fd5cc0352555ed1148548877

https://app.netlify.com/sites/using-glamor/deploys/59e64fd70b79b74e6996088b

File: www/src/pages/docs/doc-links.yaml [hunk 1]
```diff
@@ -34,4 +34,5 @@
 - title: Contributing
   links:
     How to Contribute: /docs/how-to-contribute/
+    Gatsby Style Guide: /docs/docs/gatsby-style-guide/
```
Comment (KyleAMathews) [inline www/src/pages/docs/doc-links.yaml]: The link should just be `/docs/gatsby-style-guide/`

**Comment by shannonbux** [comment]
wait really? Because at the top of the file it says /docs/docs/gatsby-style-guide.md

**Comment by KyleAMathews** [comment]
That's the path in the file system — we don't use the exact same path when constructing the website. All the other doc pages are the same /docs/docs/something.md but then on the website /docs/something/

**Pull request merged**